.github/workflows/ci.yml

```diff
@@ -103,7 +103,7 @@ jobs:
 
     - name: Lint with clippy
       shell: bash
-      if: (matrix.rust_version != 'nightly') && !contains(matrix.platform.options, '--no-default-features')
+      if: (matrix.rust_version == '1.57.0') && !contains(matrix.platform.options, '--no-default-features')
       run: cargo clippy --all-targets --target ${{ matrix.platform.target }} $OPTIONS --features $FEATURES -- -Dwarnings
```

Comment:
I feel like we should also run `clippy` on latest at least, it typically has a few new lints that we might want to clean up. MSRV should just be an "it builds" kind of test (also across all our platforms).

Reply:
The problem is that those are mutually exclusive. So passing clippy run on stable, could result in error on old stable, and vice-versa.